src/models/folders/constants.py

```diff
@@ -1,3 +1,4 @@
 __author__ = 'ibininja'
 
-COLLECTION = "folders"
\ No newline at end of file
+COLLECTION = "folders"
+TASKS_COLLECTION = "tasks"
```

Comment:
I think is best to have constants for each model; Incase in the future we add more constants specific to Task class. In this case Tasks_collection be in src/models/tasks/constants.py
If you would not mind I will make the change?